src/Font.zig

```diff
@@ -176,14 +188,13 @@ pub const FontId = enum(u64) {
             kvs_array[i] = .{ enumField.name, @field(FontId, enumField.name) };
         }
         const map = std.StaticStringMap(FontId).initComptime(kvs_array);
-        for (@typeInfo(TTFBytes).@"struct".decls) |decl| {
+        for (@typeInfo(builtin).@"struct".decls) |decl| {
             std.debug.assert(map.get(decl.name) == FontId.fromName(decl.name));
         }
     }
 };
 
-// functionality for accessing builtin fonts
-pub const TTFBytes = struct {
+pub const builtin = struct {
     pub const InvalidFontFile = "This is a very invalid font file";
```

Comment:
Why do we have this?  It's showing up in the font selection dropdowns (both in the text entry and theming part of the demo).  In main it logs errors, and in this PR it panics if you choose it.